plutarch-test/plutarch-test.cabal

```diff
@@ -118,10 +123,13 @@ executable plutarch-test
       Plutarch.RecSpec
 
   other-modules:
+    BaseSpec
```

Comment:
These should be Plutarch.{Base,Extra}Spec

Reply:
I can't do that because the `sydtest` magic stops working then, sorry.